### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by navdeepsinghkhalsa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,10 +14,10 @@
   },
   "license": "MIT",
   "dependencies": {
-    "jsdom": "^9.8.3"
+    "jsdom": "^12.2.0"
   },
   "devDependencies": {
-    "babel-cli": "^6.16.0",
-    "babel-preset-es2015": "^6.16.0"
+    "babel-cli": "^6.26.0",
```
Comment (bogas04) [inline package.json]: We can use Babel 7 now.

File: package.json [hunk 2]
```diff
@@ -14,10 +14,10 @@
...
+    "babel-preset-es2015": "^6.24.1"
```
Comment (bogas04) [inline package.json]: It's recommended to use `@babel/present-env`

File: src/index.js [hunk 3]
```diff
@@ -1,24 +1,30 @@
-'use strict';
+import { JSDOM } from 'jsdom';
 
-import dom from 'jsdom';
+const SGPC_URL = 'http://old.sgpc.net/hukumnama/sgpconlinehukamnama.asp';
 
-const SGPC_URL = `http://old.sgpc.net/hukumnama/sgpconlinehukamnama.asp`;
+module.exports = () => new Promise((resolve, reject) => {
```
Comment (bogas04) [inline src/index.js]: We can use `export default`

File: src/index.js [hunk 4]
```diff
@@ -1,24 +1,30 @@
...
+          ang: parseInt(/Page:[ ]*([\d ]+)/.exec(englishAng)[1], 10),
+          date: document.querySelector('font[face="Georgia, Times New Roman, Times, serif"]').textContent.trim().slice(1, -1),
+          url: SGPC_URL
+        });
+      } catch (e) {
+        reject(e);
+      }
+    })
+    .catch(e => reject);
+});
```
Comment (bogas04) [inline src/index.js]: If you feel like, you can convert this into a `async` function as node 8 supports it

File: package.json (hunk 1 above)
Comment (navdeepsinghkhalsa) [reply package.json]: Should we just drop Babel, since we're writing it for Node.js (as per your description)?

File: package.json (hunk 2 above)
Comment (navdeepsinghkhalsa) [reply package.json]: Same as above

File: src/index.js (hunk 3 above)
Comment (navdeepsinghkhalsa) [reply src/index.js]: Depending on if we use Babel, and if we're targeting Node 8

File: src/index.js (hunk 4 above)
Comment (navdeepsinghkhalsa) [reply src/index.js]: Achhaa Jee

File: package.json (hunk 1 above)
Comment (navdeepsinghkhalsa) [reply package.json]: @bogas04 Please let me know

Comment (bogas04) [reply package.json]: Yeah sure, we can drop babel

**Pull request merged**